packages/backend-modules/payments/lib/workers/StripeWebhookWorker.ts

```diff
@@ -95,22 +107,27 @@ export class StripeWebhookWorker extends BaseWorker<WorkerArgsV1> {
           await processChargeRefunded(ctx, job.data.company, event)
           break
         default:
-          console.log('skipping %s no handler for this event', event.type)
+          this.logger.debug(
+            { eventId: event.id, eventType: event.type },
+            'skipping webhook event; no handler for this event',
+          )
       }
     } catch (e) {
-      console.error(
-        'processing stripe event %s [%s] failed',
-        event.id,
-        event.type,
+      this.logger.error(
+        {
+          eventId: event.id,
+          eventType: event.type,
+          error: e,
+        },
+        'processing stripe event failed',
       )
-      console.error(e)
+
       throw e
     }
 
-    console.log(
-      'successfully processed stripe event %s [%s]',
-      event.id,
-      event.type,
+    this.logger.info(
```

Comment:
why is this info and not debug as for the other workers?

Reply:
To be able to see the health of stripe event processing